scipy/signal/signaltools.py

```diff
@@ -2315,15 +2315,15 @@ def resample(x, num, t=None, axis=0, window=None):
         return y, new_t
 
 
-def resample_poly(x, up, down, axis=0, window=('kaiser', 5.0)):
+def resample_poly(x, up, down, axis=0, window=('kaiser', 5.0), border='zero'):
```

Comment:
`border` is not how it's called in `filtfilt`, `padtype` would be better (unless you chose `border` because we call it that elsewhere in `scipy`)

Reply:
Happy to change the name from border to padtype. Not sure where I got border from...

Reply:
Update: it is now called padtype